### DISCUSSION TITLE
Parity when porting tools that already exist in z/OS OMVS Base

### DISCUSSION THREAD

**Discussion by lbdyck** [zopencommunity/meta#746] [General]
When porting a tool to z/OS from the open-source world that already has a IBM supplied port that is delivered as part of the z/OS base distribution it is ***mandatory*** that functionality found in the z/OS version ***not*** be lost.

Example:  `iconv` from IBM supports `-F` while the open-source port does not.

It is also important that all ports include an updated `man` page.

**Comment by IgorTodorovskiIBM** [comment]
Somewhat related conversation, discussion on how we should handle tools that conflict with /bin/ tools: https://github.com/orgs/ZOSOpenTools/discussions/438#discussioncomment-8829111

Ideally, we would want to support all options as /bin, but in some cases the community may provide improved option or different options that do the same thing, so providing a mechanism to select between /bin/ and zopen tools for particular tools may be helpful.
